### PR TITLE
Show only issued facilities

### PR DESCRIPTION
**Pull request by abhi-markan**
1. UI server function to only return issued facilities
2. Function name updated
3. Unit test case update

### CODE REVIEW

File: trade-finance-manager-api/src/v1/controllers/deal.controller.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ const submitACBSIfAllPartiesHaveUrn = async (dealId) => {
   const allRequiredPartiesHaveUrn = allPartiesHaveUrn(deal);
 
   if (allRequiredPartiesHaveUrn) {
+    console.log('*******ACBS*************');
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/controllers/deal.controller.js:93]: do we need this console.log?

File: trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js [hunk 2]
```diff
@@ -96,6 +96,7 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
        */
       const updatedDeal = await api.updateDeal(dealId, updatedDealWithTasks);
       if (dealController.canDealBeSubmittedToACBS(mappedDeal.submissionType)) {
+        console.log('*************YES***************');
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js:99]: do we need this console.log?

File: trade-finance-manager-api/src/v1/controllers/deal.controller.js (hunk 1 above)
Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/controllers/deal.controller.js]: Sorry just removed.

File: trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js (hunk 2 above)
Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js]: Removed

**Review by IgnatG — APPROVED** [review]

**Pull request merged**